src/components/entry/EntryDetails.js

```diff
@@ -81,7 +76,7 @@ const Description = styled.section`
   }
 `
 
-const EntryDetails = ({ isDesktop }) => {
+const EntryDetails = () => {
   const {
     params: { id },
   } = useRouteMatch()
```

Comment:
I know this isn't from this PR, but it might be more idiomatic to use the `useParams` hook here and avoid the deep deconstruction.

Reply:
Yes!!